release/src/main/scripts/verify_release_build.sh

```diff
@@ -123,12 +125,14 @@ if [[ ! -z `which hub` ]]; then
   # The version change is needed for Dataflow python batch tests.
   # Without changing to dev version, the dataflow pipeline will fail because of non-existed worker containers.
   # Note that dataflow worker containers should be built after RC has been built.
-  sed -i -e "s/${RELEASE_VER}/${RELEASE_VER}.dev/g" sdks/python/apache_beam/version.py
-  sed -i -e "s/sdk_version=${RELEASE_VER}/sdk_version=${RELEASE_VER}.dev/g" gradle.properties
+  sh "$SCRIPT_DIR"/set_version.sh "$RELEASE_VER"
```

Comment:
This script might also touch `buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy`. Might add a `git add` for that.

Reply:
found that `set_version.sh --git-add` can replace manual git addition